admin/server/app/createStaticRouter.js

```diff
@@ -69,13 +69,16 @@ module.exports = function createStaticRouter (keystone) {
 	/* Prepare LESS options */
 	var elementalPath = path.join(path.dirname(require.resolve('elemental')), '..');
 	var reactSelectPath = path.join(path.dirname(require.resolve('react-select')), '..');
+	var keystoneTinymcePath = path.dirname(require.resolve('keystone-tinymce'));
```

Comment:
I'm unsure why the `reactSelectPath` joins `..` to the path. The tinymce path was incorrect so I left it without